system-addon/content-src/styles/_activity-stream.scss

```diff
@@ -22,12 +21,7 @@ h2 {
 }
 
 a {
-  color: $link-primary;
```

Comment:
Why did this need to move into Base? I suppose technically there could be links under `body` that aren't under `.outer-wrapper` but there's other things in this file that are using `var()`…?

Reply:
I moved to `.outer-wrapper` because that's the first (topmost?) node that we control from react. I can't (cleanly) set classes or css vars to body or anything above `.outer-wrapper`. I positive side effect of moving the themed stuff to be sandboxed in `.outer-wrapper` is that this doesn't break snippets styles.

Reply:
I haven't actually used css variables, but looking at m-c usage/declarations, they seem to typically be under a `:root { --var: … }` selector.

@k88hudson do you have any preference or know of desired usage patterns?

Reply:
mconley points out that there is some movement to more appropriately scope css variables for performance reasons, e.g., https://bugzilla.mozilla.org/show_bug.cgi?id=1435122